### ISSUE TITLE
Cooking Mama 5: Bon Appetit! ‒ Mirrored Screen

### ISSUE THREAD

**Issue by Dragios** [citra-emu/citra#2235]
Enable hardware renderer for the first time will result in a mirrored screen. Switching to software renderer for the first time also will result in a black screen but the game audio will continue playing.
![Mirrored screen](https://cloud.githubusercontent.com/assets/18496868/20743307/99130814-b710-11e6-8679-0be88b18b1aa.png)

Log for mirrored screen:
http://pastebin.com/Mub2dwDB

Ironically, I did found a **workaround** for this issue.
The trick is to reset the game and the issue seems to fix itself. The fix is sort of permanent until you delete the save file. So although is not annoying since it is a one-time fix but still is an issue with the emulator itself. This time using software renderer will **not** result in black screen
![Normal screen](https://cloud.githubusercontent.com/assets/18496868/20743312/9fc03ec0-b710-11e6-9f8c-b17db56fc38a.png)

Disable hardware renderer (when black screen) will spam with this log:
`Render.OpenGL <Error> video_core\renderer_opengl\renderer_opengl.cpp:DebugHandler:483: API ERROR 1282: GL_INVALID_OPERATION error generated. Texture name does not refer to a texture object generated by OpenGL.`

Log after reset the game:
```
Service.DSP <Warning> core\hle\service\dsp_dsp.cpp:DSP_DSP::SetSemaphore:266: (STUBBED) called
Service.DSP <Warning> core\hle\service\dsp_dsp.cpp:DSP_DSP::SetSemaphore:266: (STUBBED) called
Service <Error> core\hle\service\service.cpp:Service::Interface::SyncRequest:74: unknown / unimplemented function 'SetClientSDKVersion': port=mic:u, cmd_buff[1]=0x50202C8
Kernel.SVC <Warning> core\hle\svc.cpp:SVC::CreateMemoryBlock:875: called addr=0x007E6000
Service.MIC <Warning> core\hle\service\mic_u.cpp:MIC_U::MapSharedMem:61: called, size=0x1000, mem_handle=0x000F802C
Service.MIC <Warning> core\hle\service\mic_u.cpp:MIC_U::GetGain:192: (STUBBED) called
Service.MIC <Warning> core\hle\service\mic_u.cpp:MIC_U::SetGain:177: (STUBBED) called, mic_gain=31
Service.FS <Error> core\file_sys\savedata_archive.cpp:FileSys::SaveDataArchive::OpenFile:53: Non-existing file C:\Users\Joshua Yip\Documents\New Folder/user/sdmc/Nintendo 3DS/00000000000000000000000000000000/00000000000000000000000000000000/title/00040000/0012d600/data/00000001/cm4dat.bin can't be open without mode create.
Service.FS <Error> core\hle\service\fs\fs_user.cpp:Service::FS::OpenFile:76: failed to get a handle for file [Wchar: /cm4dat.bin]
Service.APT <Warning> core\hle\service\apt\apt.cpp:Service::APT::AppletUtility:318: (STUBBED) called command=0x00000004, buffer1_size=0x00000001, buffer2_size=0x00000001, buffer1_addr=0x0FFFFF60, buffer2_addr=0x0FFFFF64
Service.APT <Warning> core\hle\service\apt\apt.cpp:Service::APT::AppletUtility:318: (STUBBED) called command=0x00000004, buffer1_size=0x00000001, buffer2_size=0x00000001, buffer1_addr=0x0FFFFFB8, buffer2_addr=0x0FFFFFBC
Service.FS <Warning> core\hle\service\fs\archive.cpp:Service::FS::File::SyncRequest:174: (STUBBED) File command OpenLinkFile Path: [Binary: 000000000000000000000000]
```

Tested based on the latest commit cf5b848

**OS:** _Windows 10 build 14393.447 (codename Redstone 1)_
**GPU:** _nVidia GT 610 2GB DDR3 (Asus)_
**Driver:** _Version 375.95_
**OpenGL:** _Version 4.5_

**Comment by Subv** [comment]
Could it be that the hw renderer can't bind the bottom screen texture and thus the texture keeps the already-bound top-screen image?

**Comment by Dragios** [comment]
Fixed by #3307
